openpype/settings/entities/schemas/projects_schema/schemas/schema_global_publish.json

```
@@ -197,6 +197,103 @@
+                    "type": "boolean",
+                    "key": "enabled",
+                    "label": "Enabled"
+                },
+                {
+                    "type": "list",
+                    "key": "profiles",
+                    "label": "Profiles",
+                    "object_type": {
+                        "type": "dict",
+                        "children": [
+                            {
+                                "key": "families",
+                                "label": "Families",
+                                "type": "list",
+                                "object_type": "text"
+                            },
+                            {
+                                "key": "hosts",
+                                "label": "Host names",
+                                "type": "hosts-enum",
+                                "multiselection": true
+                            },
+                            {
+                                "key": "task_types",
+                                "label": "Task types",
+                                "type": "task-types-enum"
+                            },
+                            {
+                                "key": "task_names",
+                                "label": "Task names",
+                                "type": "list",
+                                "object_type": "text"
+                            },
+                            {
+                                "key": "subsets",
+                                "label": "Subset names",
+                                "type": "list",
+                                "object_type": "text"
+                            },
+                            {
+                                "type": "boolean",
+                                "key": "delete_original",
+                                "label": "Delete Original Representation"
+                            },
+                            {
+                                "type": "splitter"
+                            },
+                            {
+                                "key": "outputs",
+                                "label": "Output Definitions",
+                                "type": "dict-modifiable",
+                                "highlight_content": true,
+                                "object_type": {
+                                    "type": "dict",
+                                    "children": [
+                                        {
+                                            "key": "output_extension",
+                                            "label": "Output extension",
+                                            "type": "text"
+                                        },
+                                        {
+                                            "key": "output_colorspace",
+                                            "label": "Output colorspace",
```

```suggestion
                                            "key": "colorspace",
                                            "label": "Colorspace",
```
Think `output` is implicit enough.